src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -228,28 +228,41 @@ export default function SnapshotListModal() {
               <Tbody>
                 {snapshotsList.map(
                   (
-                    { uuid: snap_uuid, created_at, content_counts, added_counts, removed_counts }: SnapshotItem,
+                    {
+                      uuid: snap_uuid,
+                      created_at,
+                      content_counts,
+                      added_counts,
+                      removed_counts,
+                    }: SnapshotItem,
                     index: number,
                   ) => (
                     <Tr key={created_at + index}>
                       <Td>{new Date(created_at).toUTCString()}</Td>
                       <Td>
-                        <ChangedArrows
-                          addedCount={
-                            (added_counts?.['rpm.advisory'] || 0) +
-                            (added_counts?.['rpm.package'] || 0)
-                          }
-                          removedCount={
-                            (removed_counts?.['rpm.advisory'] || 0) +
-                            (removed_counts?.['rpm.package'] || 0)
-                          }
-                        />
+                        {snapshotUUID && (
+                          <Link
+                            to={`/content/${uuid}/snapshots/${snapshotUUID}`}
+                            style={{ textDecoration: 'none' }}
+                          >
+                            <ChangedArrows
+                              addedCount={
+                                (added_counts?.['rpm.advisory'] || 0) +
+                                (added_counts?.['rpm.package'] || 0)
+                              }
+                              removedCount={
+                                (removed_counts?.['rpm.advisory'] || 0) +
+                                (removed_counts?.['rpm.package'] || 0)
+                              }
+                            />
+                          </Link>
+                        )}
                       </Td>
                       <Td>{content_counts?.['rpm.package'] || 0}</Td>
                       <Td>{content_counts?.['rpm.advisory'] || 0}</Td>
                       <Td>
-                        <RepoConfig repoUUID={uuid} snapUUID={snap_uuid}/>
-                        </Td>
+                        <RepoConfig uuid={uuid} snapUUID={snap_uuid} />
```

Comment:
no need to change the variables on this component, as mentioned on our call, you can revert this change.